src/components/Header/Header.styled.js

```diff
@@ -0,0 +1,182 @@
+import styled from 'styled-components';
+import { media } from '../Styled/mediaqueries';
+
+const StyledHeader = styled.header`
+  background-color: var(--color-main-background);
+  color: var(--font-main-white);
+  min-height: 100vh;
+  margin: 0 auto;
+  padding: 0 5rem;
+  box-sizing: border-box;
+  flex-direction: column;
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+
+  ${media.tablet`
+    padding: 0 2rem;
+    min-height: 600px;
+    font-size: 1.5rem;
+  `}
+
+  .header__interactive-canvas {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    z-index: -1;
+  }
+
+  .header__content {
+    max-width: var(--max-width);
+    margin: 0 auto;
+    padding-top: 10rem;
+
+    ${media.tablet`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 90%;
+      `}
+  }
+
+  .header__title,
+  .header__name,
+  .header__profession,
+  .header__description,
+  .header__link {
```

Comment:
Ciągle jest powtarzana nazwa `header`, może warto wykorzystać `&`, aby nie powielać tego?
np., https://stackoverflow.com/questions/72303874/should-we-use-bemblock-element-modifier-in-react-styled-component